src/app/helptext/services/components/service-netdata.ts

```diff
@@ -0,0 +1,75 @@
+import { T } from '../../../translate-marker';
+import { Validators } from '@angular/forms';
+import {
+    regexValidator
+  } from '../../../pages/common/entity/entity-form/validators/regex-validation';
+
+export default {
+    global_paratext: T('Global Settings'),
+
+    history: {
+        placeholder: T('History'),
+        tooltip: T('The number of <samp>entries</samp> the netdata daemon will\
+ by default keep in memory for each chart dimension. Default is 86400.'),
+     validation: [ Validators.required, regexValidator(/^\d+$/) ]
+    },
+    update_every: {
+        placeholder : T('Update Frequency'),
+        tooltip: T('The frequency, in seconds, for data collection.'),
+        validation: [Validators.required, regexValidator(/^\d+$/) ]
+    },
+    http_port_listen_backlog: {
+        placeholder : T('HTTP Port Listen Backlog'),
+        tooltip: T('The port backlog'),
```

Comment:
No idea what this means.  And the tooltip should be but is not a sentence.

Reply:
changed, from this source: https://www.freebsd.org/cgi/man.cgi?query=listen&sektion=2